feder/letters/models.py

```diff
@@ -43,6 +43,21 @@ def with_author(self):
     def for_milestone(self):
         return self.prefetch_related('attachment_set').with_author()
 
+    def is_not_draft(self):
+        return (self.exclude())
+
+    def is_draft(self, is_draft):
```

Comment:
Po co to przyjmuje argument?

Reply:
usunięte. lista listów miała nie zawierać szkiców, ale wycofałem to